src/components/FeaturedProducts/FeaturedProducts.test.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import '@testing-library/jest-dom/extend-expect';
+import { render, cleanup, queryByTestId } from '@testing-library/react';
+import mockFeaturedProducts from "../../utils/featured-products.json";
+import FeaturedProductsGrid from './'
+
+describe('<FeaturedProductsGrid />', () => {
```

Comment:
Good Job, but the intention of the required tests it's that you test the behavior of the whole `HomePage` page, similar to the users see the app in the browser. This test's petty good because the `FeaturedProductsGrid` is stateless so that means that from here you pass the data that you want to test.

Same comment for the others pages `Cart, Porduct List, Product Detail and Search Results`.

Reply:
Yup, this is part of the HomPage, but I missed to add HomePage itselft to the test